.golangci.yml

```diff
@@ -14,6 +14,8 @@ linters-settings:
     min-occurrences: 2
 
 linters:
-  enable-all: true
+  #enable-all: true
+  disable-all: true
```

Comment:
disable all or enable all?

Reply:
just temporary setup : golangci does not tolerate when importing packages from another branch. I will revert this to original, now that spec is merged.

Reply:
I remains an issue I don't  know how to solve: when extending the interface of a dependent pkg and trying to test the integration. I know how to twist travis, etc... but not golangci